docs/risk/risk-faq.md

```diff
@@ -16,13 +16,13 @@ Yes, EigenLayer does reuse the same economic security by enabling different AVSs
 
 Pooling allows various AVSs to contribute to a larger, shared economic security base, enhancing economic safety. This is because the cost to compromise any individual AVS increases significantly. For instance, consider 100 AVSs each with \$1B in economic security versus 100 AVSs sharing \$100B in economic security. The cost to attack any single AVS in the pooled system has effectively multiplied by 100.
 
-In addition to pooled security, EigenLayer lets each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and can only be slashed by one AVS. This arrangement guarantees AVS customers unconditional safety. 
+In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and can only be slashed by one AVS. This arrangement can provide game-theoretic guarantees for AVS customers. 
```

Comment:
> can only be slashed by one AVS

"will only be slashable by one AVS"

Reply:
> arrangement can provide game-theoretic guarantees for AVS customers

"arrangement aims to provide stronger game-theoretic guarantees on the stake backing AVSs building on EigenLayer."

Reply:
suggestion seems super reasonable. change made here https://github.com/Layr-Labs/eigenlayer-docs/pull/59/commits/9f14e709a4225702c16003a82a8c9775716ca13c